github-for-jira.sd.yml

```diff
@@ -240,7 +286,20 @@ environmentOverrides:
           instance: t2.small
           min: 1
           max: 5
-          metrics: *CpuMemScalingRules
+          metrics: *CpuMemAndQueuesScalingRules
+    resources:
+      - type: globaledge
+        name: proxy
+        attributes:
+          ip_whitelist:
+            - public
+          routes:
+            - match: # Blackhole IP that keeps spamming us
+                prefix: /
+                external_address_header_match:
+                  - 94.156.174.137
+              route:
+                cluster: blackhole # Deny by sending traffic to blackhole
```

Comment:
wasn't the reason to not have this was because we didn't want it to be publicly available and only accessible through the VPN?

Reply:
How would we install it into prod Jira then?

Reply:
I'll remove it from this PR then, so this conversation won't slow down the scaling rules change.

Reply:
:+1: